### PR TITLE
refactor: use linkcheck (for local server) to additionally check anchor links

### PR DESCRIPTION
**Pull request by emmenko**
Another try at using `linkcheck` (previous try #2476 )

### CODE REVIEW

File: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/* eslint-disable */
+//prettier-ignore
+module.exports = {
```
Comment (CarlosCortizasCT) [inline .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs:3]: Just curious about what is this plugin for 🤔

**Review by CarlosCortizasCT — APPROVED** [review]
lgtm

File: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs (hunk 1 above)
Comment (emmenko) [reply .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs:3]: To be able to use yarn commands such as `yarn workspaces foreach`.

https://github.com/yarnpkg/berry/blob/HEAD/packages/plugin-workspace-tools/README.md

I had a command using that in this PR but then changed it back. The plugin is anyway useful and we also use it in other repos.

**Review by tdeekens — APPROVED** [review]
Looks good to me. Thanks.

**Pull request merged**